client/app/domain/glue/add/domain-glue-add-or-edit.controller.js

```diff
@@ -43,7 +43,7 @@ angular.module("controllers").controller(
             if (!this.domain.glueRecordMultiIpSupported && model.ips.length > 1) {
                 valid = false;
             } else {
-                valid = !_.isEmpty(model.ips) && _.every(model.ips, (ip) => this.Validator.isValidIpv4(ip) || (_.get(this.domain, "glueRecordIpv6Supported", false) && this.Validator.isValidIpv6(ip)));
+                valid = !_.isEmpty(model.ips) && _.every(model.ips, (ip) => this.Validator.isValidIpv4(ip) || (_.get(this.domain, "glueRecordIpv6Supported", false) && this.Validator.ip.isValid(ip, { version: 6, cidr: "FORBIDDEN" })));
```

Comment:
`this.validator.isValidIpv4(ip)` no longer exists, should use `this.validator.ip.isValid(ip, { version: 4 });`